e2e/nomostest/nt.go

```diff
@@ -119,8 +119,8 @@ type NT struct {
 	DefaultWaitTimeout time.Duration
 
 	// DefaultReconcileTimeout is the default timeout for the applier to wait
-	// for object reconcilition.
-	DefaultReconcileTimeout time.Duration
+	// for object reconciliation.
+	DefaultReconcileTimeout *time.Duration
```

Comment:
Why make it a pointer? Is this to enable the `nil` check when it's unset?

Reply:
Has to allow nil to unset it, to replace the default value.

Reply:
I didn't add a test that does unset it, but I imagine it might be useful for the stress test, where 1m is too short, and we might as well stress test the default config.